src/PKSim.Core/Services/IndividualSimulationEngine.cs

```diff
@@ -122,23 +120,9 @@ private void updateResultsName(IndividualSimulation simulation)
          simulation.DataRepository.Name = simulation.Name;
       }
 
-      public Task RunForBatch(IndividualSimulation individualSimulation, bool checkNegativeValues)
-      {
-         //we want to export all 
-         return runSimulation(individualSimulation, exportAll: true, raiseEvents: false, checkForNegativeValues: checkNegativeValues);
-      }
-
-      private void updatePersistableFor(IndividualSimulation simulation, bool exportAll)
-      {
```

Comment:
the engine should not worry about was output is selected etc.. this is done in the runner now